### PR TITLE
[Dev UI] Display correctness in WB Banner

### PR DESCRIPTION
**Pull request by jeremywiebe**
## Summary:

I changed the "Check Answer" button result to be displayed in a WB banner so that it doesn't require a second click to dismiss it. Now it just disappears after a short timeout. 

image

Issue: "none"

## Test plan:

`yarn dev`
Paste some questions into the Flipbook 
Click "Check answer"

### CODE REVIEW

File: dev/flipbook.tsx [hunk 1]
```diff
@@ -204,6 +208,9 @@ function SideBySideQuestionRenderer({
 function GradableRenderer(props: QuestionRendererProps) {
     const {question, apiOptions} = props;
     const rendererRef = useRef<Renderer>(null);
+    const [score, setScore] = useState<PerseusScore | undefined>();
```
Comment (handeyeco) [inline dev/flipbook.tsx:211]:
```suggestion
    const [score, setScore] = useState<PerseusScore>();
```
Does the same thing with less code.

File: .changeset/clean-comics-stare.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/perseus-dev-ui": patch
```
Comment (handeyeco) [inline .changeset/clean-comics-stare.md:2]: Do we need a changeset for `\dev` stuff? Or is that a holdover from when it was in `\packages`?

**Review by handeyeco — APPROVED** [review]

File: dev/flipbook.tsx (hunk 1 above)
Comment (jeremywiebe) [reply dev/flipbook.tsx:211]: Weird. I thought I tried that and it complained that I wasn't providing a value. It works as you say. Thanks!

File: .changeset/clean-comics-stare.md (hunk 2 above)
Comment (jeremywiebe) [reply .changeset/clean-comics-stare.md:2]: I think our tooling will yell if I don't make a changeset for this.  We could disable it, but 🤷

Comment (handeyeco) [reply .changeset/clean-comics-stare.md:2]: Usually I wouldn't care about extra changesets, but `@khanacademy/perseus-dev-ui` isn't a think right? Like there's no `packages/perseus-dev-ui`. Seems like it'd confuse new contributors.

Comment (jeremywiebe) [reply .changeset/clean-comics-stare.md]: Going to try to mark the `package.json` as `private: true` https://docs.npmjs.com/cli/v10/configuring-npm/package-json#private  

I think that'll stop the tooling from asking for a changeset in this "package".

Comment (jeremywiebe) [reply .changeset/clean-comics-stare.md]: Ok. That worked.  We shouldn't be publishing this package anyways so I think that's the right call.

**Pull request merged**